src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -286,6 +286,21 @@ public static string GetAzureWebsiteUniqueSlotName(this IEnvironment environment
             return name?.ToLowerInvariant();
         }
 
+        /// <summary>
+        /// Gets the Antares version.
+        /// </summary>
+        public static string GetAntaresVersion(this IEnvironment environment)
+        {
+            if (environment.IsLinuxAzureManagedHosting())
+            {
+                return environment.GetEnvironmentVariableOrDefault(AntaresVersionLinux, string.Empty);
+            }
```

Comment:
@balag0 Trying to add this as part of status API. Is this the right condition and environment variable to use for linux ? Just want to double check...

Reply:
Yes, this should work right away for AppService plan. Will make the change to inject this setting for Consumption next release.

Reply:
Great, thanks !